endpoints/openrtb2/auction.go

```diff
@@ -2275,6 +2275,7 @@ func writeError(errs []error, w http.ResponseWriter, labels *metrics.Labels) boo
 		metricsStatus := metrics.RequestStatusBadInput
 		for _, err := range errs {
 			erVal := errortypes.ReadCode(err)
+			glog.Errorln("MSP Error", err.Error())
```

Comment:
The `break` below  would cause you miss errors

Reply:
done